api/bin/chainflip-cli/src/settings.rs

```diff
@@ -112,7 +112,7 @@ pub enum CliCommand {
 	)]
 	Redeem {
 		#[clap(
-			help = "Amount to redeem in FLIP (omit this option to redeem all available FLIP)",
+			help = "Amount to redeem in FLIP (omit this option to redeem all available FLIP). Only accepts up to 6 decimal places of precision.",
```

Comment:
> Only accepts up to 6 decimal places of precision.

This is not true in the current impl: we accept more than 6 and then silently ignore it...

Reply:
The user doesn't really care better to make it clear for them

Reply:
with the rounding I updated this to state that beyond 6, it's rounded.